train_management/views/calendar.py

```diff
@@ -0,0 +1,68 @@
+from django.contrib.auth.decorators import login_required
+from django.http import JsonResponse
+from django.shortcuts import render
+from django.urls import reverse
+from django.utils import dateformat, formats
+
+from train_management.models import Availability, Vehicle
+
+
+@login_required
+def calendar(request):
+    return render(request, "train_management/calendar.html")
+
+
+def get_availability_data(peter):
+    availabilities_out = []
+    title = ''
+    color = ''
+    url = ''
+    start_formatted = ''
+    end_formatted = ''
+    availabilites = Availability.objects.all()
+
+    for availability in availabilites:
+        if availability.dayplanning:
+            title = availability.dayplanning.label
+            url = ''
+            color = 'blue'
+        else:
+            start_formatted = dateformat.format(availability.start, formats.get_format('DATE_FORMAT'))
```

Comment:
Are the start and end datetime timezone aware objects? Or do you need to set the timezone?